ndbclient/ndbclient.go

```diff
@@ -48,6 +47,7 @@ func NewNDBClient(username, password, url, caCert string, skipVerify bool) *NDBC
 func (ndbClient *NDBClient) Get(path string) (*http.Response, error) {
 	url := ndbClient.url + "/" + path
 	req, err := http.NewRequest("GET", url, nil)
+	req.Header.Add("Cookie", "eraAuth=eyJhbGciOiJSUzUxMiJ9")
 	if err != nil {
 		// fmt.Println(err)
 		return nil, err
```

Comment:
Please don't check-in the header changes, these are just to circumvent the test-drive errors.

Reply:
Yes, I'll be reverting these check ins